### ISSUE TITLE
Please support Velocity

### ISSUE THREAD

**Issue by EmanYm** [GeorgH93/Minepacks#260]
### Feature request

**Feature description**
Support Velocity proxy along side Bungeecord

**How the feature is useful**
Many server networks use Velocity instead of Bungeecord or Waterfall.  I know I have Velocity set up correctly, but when I set "UseBungeeCord" to TRUE in the MinePacks config it prints this in the console at start up
```
06.05 22:58:53 [Server] WARN [Minepacks] You have BungeeCord enabled for the plugin, but it looks like you have not enabled it in your spigot.yml! You probably should check your configuration.
06.05 22:58:53 [Server] WARN [Minepacks] When using BungeeCord please make sure to set the UUID_Type config option explicitly!
06.05 22:58:53 [Server] WARN [Minepacks] You have BungeeCord enabled for the plugin, but it looks like you have not enabled it in your spigot.yml! You probably should check your configuration.
```

**Comment by GeorgH93** [comment]
Addressed with [v2.4.19](https://github.com/GeorgH93/Minepacks/releases/tag/v2.4.19).

**Comment by EmanYm** [comment]
I will grab a copy and use it right now.
You can close this ticket at any time.
